package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "gofaster-poc",
+  "title": "Web Compat",
+  "description": "Urgent post-release fixes for web compatibility.",
+
+  "license": "MPL-2.0",
+  "repository": "https://github.com/mozilla/webcompat-addon",
+  "version": "0.0.1",
+
+  "scripts": {
+    "jake": "./node_modules/.bin/jake"
+  },
+
+  "devDependencies": {
+    "eslint": "^3.4.0",
+    "eslint-plugin-json": "^1.2.0",
+    "eslint-plugin-mocha": "^4.5.1",
+    "eslint-plugin-mozilla": "0.0.3",
+    "eslint-plugin-promise": "^2.0.1",
+    "jake": "^8.0.12"
```

Comment:
I've just pushed a little change that ensures compatibility with Node 4, wich is the current LTS version. Support for v0.12 drops out in December, so 4.x is the oldest supported release. I guess we should be fine? :)